R/corplot.R

```diff
@@ -669,7 +709,7 @@ CorPairsPlot <- function(
                 diag_type = diag_type, diag_args = diag_args, layout = layout,
                 cor_method = cor_method, cor_palette = cor_palette, cor_palcolor = cor_palcolor,
                 cor_size = cor_size, cor_format = cor_format, cor_fg = cor_fg, cor_bg = cor_bg, cor_bg_r = cor_bg_r,
-                theme = theme, theme_args = theme_args, palette = palette, palcolor = palcolor,
+                theme = theme, theme_args = theme_args, palette = palette[nm], palcolor = palcolor[nm],
```

Comment:
Should it be `palette[[nm]]` and `palcolor[[nm]]`? Same for the other ones in this file.